core/src/net/sf/openrocket/util/TestRockets.java

```diff
@@ -1629,5 +1634,84 @@ public static OpenRocketDocument makeTestRocket_for_estimateFileSize() {
 		rocket.enableEvents();
 		return rocketDoc;
 	}
+
+	// the following two models are used in testing
+	// otherwise-identical rockets, one created in the obvious way
+	// using a single finset and the other creating three pods, each
+	// with a single fin.
+	public static final Rocket make3FNCNoPods()  {
```

Comment:
this sounds almost identitcal to the `makeAlphaIII` function.  Is it possible to combine these? 

(where the *withPods function would correspond to a pod-ized version of the AlphaIII?)

Reply:
I'll take a look.